src/3p-frame.js

```diff
@@ -202,22 +201,18 @@ export function addDataAndJsonAttributes_(element, attributes) {
 /**
  * Get the bootstrap script URL for iframe.
  * @param {string} type
- * @param {!Window} win
  * @return {string}
  */
-export function getBootstrapUrl(type, win) {
+export function getBootstrapUrl(type) {
   if (getMode().localDev || getMode().test) {
     const filename = getMode().minified
       ? `./vendor/${type}.`
       : `./vendor/${type}.max.`;
     return IS_ESM ? filename + 'mjs' : filename + 'js';
   }
-  if (isExperimentOn(win, '3p-vendor-split')) {
-    return IS_ESM
-      ? `${urls.thirdParty}/${internalRuntimeVersion()}/vendor/${type}.mjs`
-      : `${urls.thirdParty}/${internalRuntimeVersion()}/vendor/${type}.js`;
-  }
-  return `${urls.thirdParty}/${internalRuntimeVersion()}/f.js`;
+  return IS_ESM
+    ? `${urls.thirdParty}/${internalRuntimeVersion()}/vendor/${type}.mjs`
+    : `${urls.thirdParty}/${internalRuntimeVersion()}/vendor/${type}.js`;
```

Comment:
nit for clarity: Consider setting an `extension` temp var, and use it on both locations. Closure will properly merge the statements for both codepaths.

```js
const extension = IS_ESM ? '.mjs' : '.js';
```

Reply:
Thank you. Updated PR.